bgfx.idl

```diff
@@ -1164,87 +1164,89 @@ func.calcTextureSize
 
 --- Create texture from memory buffer.
 func.createTexture
-	"TextureHandle"               --- Texture handle.
-	.mem   "const Memory*"        --- DDS, KTX or PVR texture binary data.
-	.flags "uint64_t"             --- Texture creation (see `BGFX_TEXTURE_*`.), and sampler (see `BGFX_SAMPLER_*`)
-	                              --- flags. Default texture sampling mode is linear, and wrap mode is repeat.
-	                              --- - `BGFX_SAMPLER_[U/V/W]_[MIRROR/CLAMP]` - Mirror or clamp to edge wrap
-	                              ---   mode.
-	                              --- - `BGFX_SAMPLER_[MIN/MAG/MIP]_[POINT/ANISOTROPIC]` - Point or anisotropic
-	                              ---   sampling.
-	.skip  "uint8_t"              --- Skip top level mips when parsing texture.
-	.info  "TextureInfo*" { out } --- When non-`NULL` is specified it returns parsed texture information.
+	"TextureHandle"                            --- Texture handle.
+	.mem   "const Memory*"                     --- DDS, KTX or PVR texture binary data.
+	.flags "uint64_t"                          --- Texture creation (see `BGFX_TEXTURE_*`.), and sampler (see `BGFX_SAMPLER_*`)
+	 { "BGFX_TEXTURE_NONE|BGFX_SAMPLER_NONE" } --- flags. Default texture sampling mode is linear, and wrap mode is repeat.
```

Comment:
It should be `{ default = "BGFX_TEXTURE_NONE|BGFX_SAMPLER_NONE" }`

Reply:
Fixed.